packages/lab/src/list-next/ListItemNext.css

```diff
@@ -0,0 +1,59 @@
+/* Default variables applied to the root element */
+.saltListItemNext {
+  --listItemNext-text-color: var(--salt-text-primary-foreground);
+  --listItemNext-background: none;
+}
+
+.saltListItemNext {
+  color: var(--listItemNext-text-color);
+  background: var(--listItemNext-background);
+  height: var(--salt-size-stackable, auto);
+  padding-inline: var(--salt-size-unit);
+  display: flex;
+  gap: var(--salt-size-unit);
+  position: relative;
+  align-items: center;
+  white-space: nowrap;
+  transition: background var(--salt-duration-perceptible) ease-in-out;
+}
+
+.saltListItemNext > * {
+  pointer-events: none;
```

Comment:
I'm going to guess, but this is probably not what you would want. What was the use-case here?